docs/learn/learn-nomination-pools.md

```diff
@@ -207,10 +225,62 @@ balance.
   all other members have left. Once they leave by withdrawing, the pool is fully removed from the
   system.
 - Nominator: Can select the validators the pool nominates.
-- State-Toggler: Can change the pool’s state and kick (permissionlessly unbond/withdraw) members if
-  the pool is blocked.
-- Root: Can change the nominator, state-toggler, or itself. Further, it can perform any of the
-  actions the nominator or state-toggler can.
+- Bouncer: Can change the pool’s state and kick (permissionlessly unbond/withdraw) members if the
+  pool is blocked.
+- Root: Can change the nominator, bouncer, or itself. Further, it can perform any of the actions the
+  nominator or bouncer can.
+
+### Pool Commissions
+
+:::info Live on Kusama
```

Comment:
FYI, this referendum will enable a `globalMaxCommission` of 10% on Polkadot and enable commission: https://polkadot.polkassembly.io/referenda/55